oviewer/prepare_draw.go

```diff
@@ -120,18 +120,19 @@ func (root *Root) setAlignConverter() {
 		maxWidths, addRight = m.maxColumnWidths(maxWidths, addRight, ln)
 	}
 
-	if !reflect.DeepEqual(m.alignConv.maxWidths, maxWidths) {
+	if !slices.Equal(m.alignConv.maxWidths, maxWidths) {
```

Comment:
Based of the fact the if starts here and ends with the function, you could do this


```suggestion
	if slices.Equal(m.alignConv.maxWidths, maxWidths) {
		return
	}
```

Reply:
Oops, that's right.